ThunderCloud/CollectionCell.swift

```diff
@@ -219,8 +222,8 @@ extension CollectionCell : UICollectionViewDelegateFlowLayout {
         guard let items = items, items.indices.contains(indexPath.item) else { return .zero }
         
         let item = items[indexPath.item]
-        let size = CollectionItemViewCell.size(for: item)
-        return CGSize(width: size.width, height: maxHeight)
+        let size = CollectionItemViewCell.size(for: item, includingVerticalPadding: false)
```

Comment:
Previously all the cells were the same height, is this no longer the case now then? 😄

Reply:
No, it should be fine for them all to vary :D AFAIK collection views will pin the cells to the top rather than centering them vertically so it should be fine to have variable heights... it means we can reduce computation slightly as we can remove the `maximumHeight` property

Reply:
Yeah that's true I believe 😄